doc/src/docs/EngineeringReference/13f-EncyclopaedicRefs.md

```diff
@@ -544,16 +544,12 @@ EvapHeatLossRegComf = 0.42\*(IntHeatProd - ActLevelConv)
 
 EvapHeatLossRegComf = 0.0
 
-EvapHeatLossDiff = 0.4148\*(SkinComfVpress - VapPress)
+EvapHeatLossDiff = 0.00305\*(5733.0 - 6.99\*IntHeatProd - VapPress)
 
 EvapHeatLoss = EvapHeatLossRegComf + EvapHeatLossDiff
 
 Where,
 
-0.68 is the passive water vapor diffusion rate, (g/h·m<sup>2</sup>·Torr)
-
-0.61 is the latent heat of water, (W·h/g)
-
 P<sub>sk</sub> is the saturated water vapor pressure at the skin temperature required to achieve the thermal comfort
```

Comment:
Ahh, delete from <tr> to </tr> to delete a row in the table. That should make it pretty easy to do.